sycl/test-e2e/Matrix/XMX8/joint_matrix_opt_kernel_feature.cpp

```diff
@@ -0,0 +1,14 @@
+// REQUIRES: matrix-xmx8
```

Comment:
Do we have a test which specifically targets unsupported hardware to check "joint_matrix is not supported at all" exception?

Reply:
Added joint_matrix_opt_kernel_feature_unsupported_hw.cpp test